frontend/src/my-books/ShelfModal.css

```diff
@@ -64,26 +65,20 @@ If not, see <https://www.gnu.org/licenses/>.
     width: 100px;
 }
 
-@media screen and (max-width: 959px) {
-    .shelfInput > .MuiOutlinedInput-root {
-        width: 100%;
-        margin-right: 600px;
-        border-radius: 20px;
-    }
-}
-
-@media screen and (max-width: 420px) {
+@media screen and (max-width: 600px) {
     .shelf-modal-container {
         text-align: center;
     }
     .shelf-button-container {
         display: flex;
         flex-direction: column-reverse;
     }
+    .shelf-button-container > .MuiButton-containedPrimary { 
+        margin-bottom:14px; 
+    }
 
     .shelf-modal-button {
         width: 100%;
-        /* override the default margin-top:0px of material-ui buttons */
-        margin-top: 24px !important;
```

Comment:
It looks like you only removed the comment here, please remove line 87 as well. 👍

Reply:
My bad, you got this one! 👍  Please ignore :)